projects/S2_Climate_health_analysis/convert_json_to_dicts.py

```diff
@@ -1,33 +1,39 @@
-#%%
+"""
```

Comment:
I thought module-level docstrings might help to give a description of the exploration scripts although my description is probably very far off the mark!

Reply:
👍